domain/miningmanager/mempool/mempool.go

```diff
@@ -632,7 +621,6 @@ func (mp *mempool) maybeAcceptTransaction(tx *consensusexternalapi.DomainTransac
 		}
 	}
 
-	// TODO: How to get fee/mass?
```

Comment:
You also need to uncomment and modify the code below : `if txFee == 0....`

Reply:
This is covered by the minRelayFee, will remove it